packages/at_commons/lib/src/verb/syntax.dart

```diff
@@ -128,14 +128,10 @@ class VerbSyntax {
   static const notifyRemove = r'notify:remove:(?<id>[\w\d\-\_]+)';
   static const enroll =
       r'^enroll:(?<operation>request|approve|deny|revoke|list)'
-      // Add colon only when operation is request, approve, deny or revoke. Do not add for list
-      r'(?:(?<=request|approve|deny|revoke):)?'
-      r'(?:(?:enrollmentId:(?<enrollmentId>[a-zA-Z0-9-]+))|'
-      r'(?:appName:(?<appName>[a-zA-Z0-9]+):)?'
-      r'(?:deviceName:(?<deviceName>[a-zA-Z0-9_]+):)?'
-      r'(?:namespaces:\[(?<namespaces>[a-zA-Z0-9;_,]+)\]:)?'
-      r'(?:totp:(?<totp>[0-9]+):)?'
-      r'(?:apkamPublicKey:(?<apkamPublicKey>.*)))?$';
+      // Positive lookbehind - Matches the value of operation named group
+      //  To ignore ":" if operation is list
```

Comment:
To be precise, this is actually to require ":" if operation is request|approve|deny|revoke